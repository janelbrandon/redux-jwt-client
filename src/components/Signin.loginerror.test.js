import React from 'react'
import renderer from 'react-test-renderer'
import Signin from './SignIn'

describe('Signin Snapshot', () => {
    it('Should render as expected with login error', () => {
        const tree = renderer.create(
            <Signin handleSignIn={() => { }} loginError="login error" />
        )
        expect(tree.toJSON()).toMatchSnapshot()
    })

})