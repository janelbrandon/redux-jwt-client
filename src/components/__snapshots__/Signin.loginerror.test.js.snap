// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signin Snapshot Should render as expected with login error 1`] = `
<form
  onSubmit={[Function]}
>
  <p>
    login error
  </p>
  <label>
    Email: 
    <input
      name="email"
      type="email"
    />
  </label>
  <br />
  <label>
    Password: 
    <input
      name="password"
      type="password"
    />
  </label>
  <br />
  <button
    className="sc-bdVaJa lmpxVR"
    type="submit"
  >
    Login
  </button>
</form>
`;
